packages/flutter_test/lib/src/test_text_input.dart

```diff
@@ -8,6 +8,7 @@ import 'dart:typed_data';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/foundation.dart';
+import 'package:pedantic/pedantic.dart';
```

Comment:
@jonahwilliams is this ok?  It's for `unawaited`.

Reply:
This would introduce a new (direct) dependency on package:pedantic. I don't think we want that.

Reply:
Removing this, but FWIW it looks like we directly depend on it elsewhere (although our pubspec doesn't indicate that).